### PR TITLE
IT VDT: Remove Debian Stretch Support

### PR DESCRIPTION
**Pull request by Deblintrake09**
|Related issue|
|-------------|
| Closing #3167  |

## Description

This PR aims to fix  remove the support for Debian Stretch in the VDT suite from the repository.

### Removed

- test cases for the following tests:
   - test_feeds/test_download_feeds
   - test_feeds/test_validate_feed_xml_content
   - test_general_settings/test_retry_interval
   - test_providers/test_disabled
   - test_providers/test_enabled
   - test_providers/test_os

---

## Testing performed

| Tester             | Test path | Jenkins | Local  | OS | Commit | Notes                |
|--------------------|-----------|---------|--------|-----|--------|----------------------|
| @Deblintrake09 (Developer)  |  test_vulnerability_detector/         | [:green_circle:](https://ci.wazuh.info/view/Tests/job/Test_integration/30588/) [:green_circle: ](https://ci.wazuh.info/view/Tests/job/Test_integration/30589/) [:green_circle:](https://ci.wazuh.info/view/Tests/job/Test_integration/30590/) | [:green_circle:](https://github.com/wazuh/wazuh-qa/files/9311166/R1.zip)  |  Centos8   |   [4524430](https://github.com/wazuh/wazuh-qa/pull/3172/commits/45244301d51e515166fb0aab74d6516050f94a46)    | Nothing to highlight |
| Seyla (Reviewer)   | test_vulnerability_detector/         | [🟢](https://ci.wazuh.info/view/Tests/job/Test_integration/30623/) [🟢](https://ci.wazuh.info/view/Tests/job/Test_integration/30624/) [🟢](https://ci.wazuh.info/view/Tests/job/Test_integration/30625/)   | ⚫  | Centos8   |   [4524430](https://github.com/wazuh/wazuh-qa/pull/3172/commits/45244301d51e515166fb0aab74d6516050f94a46)    | Nothing to highlight |

### CODE REVIEW

**Review by damarisg — CHANGES_REQUESTED** [review]
It is required to add a description in the CHANGELOG file.

**Review by damarisg — APPROVED** [review]

File: tests/integration/test_vulnerability_detector/test_feeds/data/test_cases/cases_validate_xml_feed_content.yaml [hunk 1]
```diff
@@ -1,208 +1,198 @@
...
-    provider_name: 'Debian Jessie'
-    expected_format: 'xml'
-    path: '/tmp/oval-definitions-jessie.xml'
-    extension: 'xml'
-    url: 'https://www.debian.org/security/oval/oval-definitions-jessie.xml'
+    provider_name: Debian Wheezy
+    expected_format: xml
+    path: /tmp/oval-definitions-wheezy.xml
+    extension: xml
+    url: https://www.debian.org/security/oval/oval-definitions-wheezy.xml
```
Comment (jmv74211) [inline tests/integration/test_vulnerability_detector/test_feeds/data/test_cases/cases_validate_xml_feed_content.yaml:128]: Wheezy and Jessie are deprecated, please remove them from all tests

**Review by jmv74211 — CHANGES_REQUESTED** [review]

File: tests/integration/test_vulnerability_detector/test_feeds/data/test_cases/cases_validate_xml_feed_content.yaml (hunk 1 above)
Comment (Deblintrake09) [reply tests/integration/test_vulnerability_detector/test_feeds/data/test_cases/cases_validate_xml_feed_content.yaml]: Fixed in https://github.com/wazuh/wazuh-qa/pull/3172/commits/dafc1ca26541ede97f9cf75e9f12a5fdc56796f0

File: CHANGELOG.md [hunk 2]
```diff
@@ -29,6 +29,7 @@ Release report: TBD
 
 ### Changed
 
+- VDT ITs: Remove Debian Stretch test support. ([#3172](https://github.com/wazuh/wazuh-qa/pull/3172)) \- (Tests)
```
[the remark this answers is not in the document]
Comment (Deblintrake09) [reply CHANGELOG.md]: Fixed in https://github.com/wazuh/wazuh-qa/pull/3172/commits/3619a281bbcd0434c06eb94d83317b247e9c740f

**Review by jmv74211 — APPROVED** [review]

**Pull request merged**